src/main/java/nextstep/qna/domain/BaseModel.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.qna.domain;
+
+import java.time.LocalDateTime;
+
+public abstract class BaseModel {
+
+    public LocalDateTime createdDate = LocalDateTime.now();
+
+    public LocalDateTime updatedDate;
+
+    public boolean deleted = false;
```

Comment:
```suggestion
    private boolean deleted = false;
```
그럼 상태 값 변경은 어떻게 하나?

```java
    protected void deleted() {
        this.deleted = true;
    }
```
아래와 같이 protected 메서드를 제공하고 자식 클래스에서 호출할 수 있도록 한다.